lib/views/help/how.html.erb

```diff
@@ -13,27 +13,73 @@
     This can also happen if a document has not been updated for some time, and the requester asks for the latest version.
-   This means that some of the information that is published on the site will already be out of date on the day it is published.
-   We never remove material from the site just because it is old or because a newer version of the material exists.
+    This means that some of the information that is published on the site will already be out of date on the day it is published.
+    We never remove material from the site just because it is old or because a newer version of the material exists.
     This is because we are trying to create a permanent archive.
   </p>
+ <h2 id="we_dont_provide_professional_advice">
+    We don't provide professional advice
+    <a href="#we_dont_provide_professional_advice">#</a>
+  </h2>
+  <p>
+    WhatDoTheyKnow contains information on a wide variety of topics. While some of the documents and files
+    published on the site contain recommendations and guidance, it is important to remember that this
+    information may be <a href="#we_dont_check_the_information">incorrect</a> or 
+    <a href="#information_may_be_out_of_date">out of date</a>. Even where information is up to date and 
+    accurate, it may not be suitable for your specific circumstances 
+    or the circumstances of someone you are trying to help. The information on the site is not 
+    intended to be used as professional advice by users of the site or anyone else. In addition, the 
+    team that runs the site cannot provide financial, medical or legal advice, or any other type of
+    professional advice. See also the help text below relating to 
+    <a href="#medical_information">medical information</a> on the site.
+  </p>
+  <p>
+    If you need advice, please consider seeking advice from a qualified professional.
+  </p>
+<h2 id="medical_information">
+    Do not rely on any medical information you find on the site
+    <a href="#medical_information">#</a>
+</h2>
+  <p>
+    WhatDoTheyKnow contains information on a wide variety of topics including information relating 
+    to healthcare. The information on the site may be 
+    <a href="#we_dont_check_the_information">incorrect</a> or 
+    <a href="#information_may_be_out_of_date">out of date</a>. 
+    Even if you find information on the site about healthcare that you believe is accurate and up to date, 
+    it may not apply to you or someone you are trying to help, even the symptoms appear to be similar.
+    In addition, the team that runs the site cannot provide medical advice or any other type of 
+    <a href="#we_dont_provide_professional_advice">professional advice</a>.
+  </p>
+  <p>
+    If you think that you may need medical advice or treatment, you should contact a qualified
+    medical professional such as a doctor or pharmacist. In the UK, you also have the option to contact the 
+    NHS advice service: 
```

Comment:
Is a comma or colon most suitable at the end of the line?

Reply:
Thank you. I have changed to a comma.